src/bitcoin-cli.cpp

```diff
@@ -467,8 +467,8 @@ class NetinfoRequestHandler : public BaseRequestHandler
         if (!batch[ID_NETWORKINFO]["error"].isNull()) return batch[ID_NETWORKINFO];
 
         const UniValue& networkinfo{batch[ID_NETWORKINFO]["result"]};
-        if (networkinfo["version"].getInt<int>() < 209900) {
-            throw std::runtime_error("-netinfo requires bitcoind server to be running v0.21.0 and up");
+        if (networkinfo["version"].getInt<int>() < 239900) {
```

Comment:
In e6603e50f4fbeeeb5a0dfd25f78b83b85d542984:
> This change is because the getpeerinfo "relaytxes" field was changed
between v23 and v24 to become optional

If we end up bumping/changing this anyway, because of the `relaytxes` change, do we actually need to revert `minfeefilter` to being non-optional?

Reply:
The idea is to maintain the current released API interface and avoid potentially breaking user space without a compelling reason (`minfeefilter` was added to getpeerinfo in 2018 in https://github.com/bitcoin/bitcoin/commit/5778bf95d94f4d187451a72a5b86ba4daaafe4ab and I believe it has had the same behavior since then for around 7 releases. So far, `-netinfo` doesn't use it).